### ISSUE TITLE
[Accessibility Audit] Search: Search Contexts Management

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34191]
### Use cases to replicate

Taken from the [user journey list document](https://docs.google.com/document/d/1IA90OFew5MTcxFndKS2Xn4vb8sOt68sj537P7AGj_Tk/edit?usp=sharing). Use this for further context.

Please take these following use cases and replicate them when running your audit.

- On any search page, I can open the search contexts dropdown and click on "Manage context” to open the search contexts management page.
- Anywhere in Sourcegraph, I can use the “Code Search” > “Contexts” menu item to open the search contexts management page.
- On the search contexts management page, I can view a list of all the contexts I have access to; I can sort and filter the list to help find what I need if the list is very long. For each context, I can see its name, description, query (if it is query-based), number of repositories (if it is repository based), and when it was last updated. I can click on the context’s name to open the context’s page.
- I can create a new search context by clicking on the “Create search context” button. I can select myself, one of my org, or (if I'm an admin) the global scope as the owner of the context. I can give the context a name and a description, and select whether the context should be public or private. I can use either a special search query or a JSON with a list of repositories and revisions to create the context; for either of these, I get autocomplete suggestions, syntax highlighting, and validation.
- I can edit a context by clicking on its name and then clicking on “Edit”. I can change anything about the context in the edit view, except the owner. I can also delete the context.

### How to audit

Follow the instructions here: [Auditing a user journey](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#auditing-a-user-journey)

**Note:** We are planning on arranging thorough third-party audit in the future, so our focus here is just to find the _majority_ of accessibility issues. Don't worry if you feel you haven't found 100% of potential issues, it is better to focus on the core essentials to complete the journey rather than spending lots of time going through every possible problem.

### How to raise discovered problems

Follow the instructions here: [Raising an accessibility bug](https://docs.sourcegraph.com/dev/background-information/web/accessibility/how-to-audit#raising-a-bug)

### Any other questions?

See the [audit FAQ](https://docs.google.com/document/d/1nCi7TvYRpj9eG0_QtgqK1c4L28EiJKVuxyuvg1lAGgo/edit#) - add your question if it isn't there already!

**Comment by limitedmage** [comment]
Audit completed with the following successor issues:
- https://github.com/sourcegraph/sourcegraph/issues/34726
- https://github.com/sourcegraph/sourcegraph/issues/34730
- https://github.com/sourcegraph/sourcegraph/issues/34740
- https://github.com/sourcegraph/sourcegraph/issues/34741
